tests/functional/auth_test.php

```diff
@@ -101,7 +101,7 @@ public function test_board_auth_oauth_setting()
 		$this->assertStringContainsString($this->lang('AUTH_PROVIDER_OAUTH_SERVICE_GOOGLE'), $crawler->filter('h3')->text());
 		$form = $crawler->selectButton($this->lang('UCP_AUTH_LINK_LINK'))->form();
 		$crawler = self::submit($form);
-		$this->assertStringContainsString('Google Accounts', $crawler->filter('title')->text());
+		$this->assertStringContainsString('Google', $crawler->filter('title')->text());
```

Comment:
This was changed to the following on master:
`$this->assertStringContainsString('accounts.google.com', $crawler->filter('base')->attr('href'));`
I think that should work as well and would not depend on the actual content of the page.